python/ray/tune/sync_client.py

```diff
@@ -199,13 +209,8 @@ def set_logdir(self, logdir):
         self._closed = False
 
     def _get_logfile(self):
-        if self._closed:
-            raise RuntimeError(
```

Comment:
@richardliaw I need some context here. Is this expected not to happen (or rather should we act on this case)? See [comment](https://github.com/ray-project/ray/issues/17506#issuecomment-914430196). We call sync_down after the client is closed, which seems contradictory to the implied semantics here. So updated the API contract. Let me know if you have a concern. Thanks!

Reply:
Yeah, we shouldn't call sync_down after the client is closed.